assets/js/googlesitekit/datastore/user/authentication.test.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * core/user Data store: Authentication info tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import {
+	createTestRegistry,
+	muteConsole,
+	subscribeUntil,
+	unsubscribeFromAll,
+} from 'tests/js/utils';
+import { STORE_NAME } from './constants';
+import './index';
+
+describe( 'core/user authentication', () => {
+	const responseAuthenticated = { authenticated: true };
```

Comment:
Please define this closer to the assertions. It's only one line, so even if it's added to every test, that's okay.

Reply:
@aaemnnosttv these values don't change, I'm not sure it's a good idea to repeat the variable in multiple places. Are you concerned about readability here?